openms-wiki/users-quickstart-guide.md

```diff
@@ -0,0 +1,8 @@
+Users Quick Start Guide
+=======================
+
+Download the current stable version of OpenMS from the [OpenMS download site](http://open-ms.de/downloads) or via the [OpenMS docker containers]().
```

Comment:
installation has docker containers link(s).

Reply:
sorry I'm not sure what you mean

Reply:
https://staging-openms.readthedocs.io/en/staging/docs/installations/installation-on-gnu-linux.html#run-via-a-bio-docker-image